source/errors/DOMException.ts

```diff
@@ -0,0 +1,10 @@
+// TODO: Use `??` instead of '||' when targeting Node.js 14
+const DOMException = globalThis.DOMException || Error;
+
+export default DOMException;
```

Comment:
The other files export a class but this time it isn't actually used outside of this file. Not sure if the function name is specific enough either. What do you think?

Reply:
It's fine 👍